accel/tcg/cputlb.c

```diff
@@ -1662,6 +1662,7 @@ int probe_access_flags(CPUArchState *env, target_ulong addr,
     return flags;
 }
 
+QEMU_ALWAYS_INLINE
```

Comment:
QEMU_ALWAYS_INLINE without `static inline` is a bit dodgy and will warn with GCC. Does this need another level of wrapping?

Reply:
Ah good point, clang did the "right" thing here, so I assumed it would be fine with GCC too. Will update to add another wrapper.